### ISSUE TITLE
Design : Optimise for desktop UI

### ISSUE THREAD

**Issue by Yzia** [mandelmonkey/bookoforbs#5]
Chrome Version 62.0.3202.94 (Build officiel) (64 bits)

Hi, please find below some points that could help to make your website a bit more user-friendly

Menu 
The menu at the below of the page is a bit weird for a website design.

Move back
When you click on a asset to see all its details, it's weird to have to click on the button "back" placed on the top left.  First thing I try to do is to click all around to move back. I didn't understand directly that you had to click on this button to move back.

**Comment by mandelmonkey** [comment]
So actually the app is meant to be a mobile app, it was built as a reaction to the iOS app being removed from the store, hence the UI is meant for mobile touch screen not desktop

I do agree we should change it to make a desktop UI at some point so I will keep this open

**Comment by mandelmonkey** [comment]
added the click to close

**Comment by Yzia** [comment]
Hi,

Another suggestion, the amount in dollar/chf/eur etc ...  into this screen is too close of the bitcrystals amount. For more visibility it would be nice to include more space between the 2.

![image](https://user-images.githubusercontent.com/34238756/33835798-e8bf9b80-de87-11e7-9373-129f44dba355.png)

**Comment by mandelmonkey** [comment]
yes I agree, the desktop version does look a bit out of place, but Ill set this as low priority as I beleive shaban wants to focus on the mobile experience with in the jiko app
